### PR TITLE
Project folders update

### PR DESCRIPTION
**Pull request by F2X**
Hi, I've done some refactoring to make autoupdate on cdnjs working (my submisson is pending). I needed built ".js" files to be onto a simple folder, so I moved result of gulp process onto "bin" folder and sources on "src".

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -2,31 +2,46 @@ var gulp = require('gulp-param')(require('gulp'), process.argv),
...
+//    .pipe(uglify())
+//    .pipe(gulp.dest('dist/'));
+//});
+//
+//gulp.task('css', function() {
+//  return gulp.src(['src/hand.css.js'])
+//    .pipe(rename({suffix: '.min'}))
+//    .pipe(uglify())
+//    .pipe(gulp.dest('dist/'));
+//});
```
Comment (deltakosh) [inline gulpfile.js]: Why commenting this?

Comment (F2X) [inline gulpfile.js]: It was duplicated from the default task. I haven't thinked that you are maybe using them in another context, I can amend my pull request to remove these changes, it's not an issue for the cdn. The only change needed is to have built and source files separated to distribute only built ones.

Comment (deltakosh) [inline gulpfile.js]: I want to allow people to get only the base version that's why :)

Comment (F2X) [inline gulpfile.js]: I've amended my pull request to revert this change
